games/dicee_dare.js

```diff
@@ -0,0 +1,216 @@
+const player = "p";
+const d1 = "H";
+const d2 = "T";
+const d3 = "C";
+const d4 = "P";
+const d5 = "L";
+const d6 = "h";
+const s = "E";
+let gameStarted = false;
+
+
+setLegend(
+  [player, bitmap`
+................
+..000000000000..
+.02222222222220.
+.02233333332220.
+.02232222233220.
+.02232222223220.
+.02233222223220.
+.02222222223220.
+.02222233333220.
+.02222232222220.
+.02222222222220.
+.02222232222220.
+.02222222222220.
+..000000000000..
+................
+................`],
+  [d1, bitmap`
+................
+..000000000000..
+.02222222222220.
+.02222222222220.
+.02222222222220.
+.02222222222220.
+.02222233222220.
+.02222233222220.
+.02222233222220.
+.02222222222220.
+.02222222222220.
+.02222222222220.
+.02222222222220.
+..000000000000..
+................
+................`],
+  [d2, bitmap`
+................
+..000000000000..
+.02222222222220.
+.02222222222220.
+.02222222222220.
+.02222222222220.
+.02233222233220.
+.02233222233220.
+.02222222222220.
+.02222222222220.
+.02222222222220.
+.02222222222220.
+.02222222222220.
+..000000000000..
+................
+................`],
+  [d3, bitmap`
+................
+..000000000000..
+.02222222222220.
+.02222222222220.
+.02222222222220.
+.02222222222220.
+.02332233223320.
+.02332233223320.
+.02222222222220.
+.02222222222220.
+.02222222222220.
+.02222222222220.
+.02222222222220.
+..000000000000..
+................
+................`],
+  [d4, bitmap`
+................
+..000000000000..
+.02222222222220.
+.02222222222220.
+.02332222223320.
+.02332222223320.
+.02222222222220.
+.02222222222220.
+.02222222222220.
+.02332222223320.
+.02332222223320.
+.02222222222220.
+.02222222222220.
+..000000000000..
+................
+................`],
+  [d5, bitmap`
+................
+..000000000000..
+.02222222222220.
+.02222222222220.
+.02332222223320.
+.02332222223320.
+.02222222222220.
+.02222233222220.
+.02222233222220.
+.02332222223320.
+.02332222223320.
+.02222222222220.
+.02222222222220.
+..000000000000..
+................
+................`],
+  [d6, bitmap`
+................
+..000000000000..
+.02222222222220.
+.02332222223320.
+.02332222223320.
+.02222222222220.
+.02332222223320.
+.02332222223320.
+.02222222222220.
+.02332222223320.
+.02332222223320.
+.02222222222220.
+.02222222222220.
+..000000000000..
+................
+................`],
+  [s, bitmap`
+................
+..000000000000..
+.02222222222220.
+.02222222222220.
+.02233222233220.
+.02233222233220.
+.02222222222220.
+.02322222222320.
+.02332222223320.
+.02333333333320.
+.02233333332220.
+.02222333332220.
+.02222222222220.
+..000000000000..
+................
+................`]
+);
+
+setSolids([]);
+let level = 0;
+const levels = [
+  map`
+.p
+..`
+];
+
+setMap(levels[level]);
+setPushables({
+  [player]: []
+});
+
+onInput("s", () => {
+
+    if (!gameStarted) {
+    console.log("Game started! Press 's' to roll the dice and 'd' to reset.");
```

Comment:
@Antime02 could you use addText() instaed of console.log?

Reply:
how do I edit the code and resubmit it like do I have to Close the pull Request and then open it again

Reply:
Sorry for the delay! nope, go to `Files changed` > `...` > `edit`

Reply:
sorry Lucas but where do I find this "File changed"